app/src/main/res/values/strings.xml

```diff
@@ -789,5 +789,6 @@ You only need to do this once, until the next time you select a new location for
     <string name="protocol_ftp" translatable="false">FTP</string>
     <string name="protocol_ftps">Secure FTP</string>
     <string name="unavailable">Unavailable</string>
+    <string name="cloud_error_fdroid">Amaze Cloud plugin not supported with Fdroid build. Please use Play Store version of Amaze.</string>
```

Comment:
"Fdroid" -> "F-Droid"
"Amaze Cloud plugin" -> "Amaze Cloud Plugin"

"... FDroid version." sounds more user friendly to me

Reply:
... also, can we show the Toast before sign in with Google dialog? (fail fast?)

Reply:
no, that'll add complication without any need.